integration-tests/kafka/pom.xml

```diff
@@ -230,6 +256,7 @@
                                     <enableHttpUrlHandler>true</enableHttpUrlHandler>
                                     <graalvmHome>${graalvmHome}</graalvmHome>
                                     <enableAllSecurityServices>true</enableAllSecurityServices>
+                                    <reportErrorsAtRuntime>true</reportErrorsAtRuntime>
```

Comment:
I don't think we want that enabled.

Reply:
Without it the native build image creation fails die to Avro generated classes. ( Even without a native test for Avro). Any idea how I can get around it ?

Reply:
As discussed with @cescoffier, we moved all Avro stuff to a separate module.